single-one-column.php

```diff
@@ -32,27 +33,38 @@
 	$show_menu = '';
 	$ancestors = get_post_ancestors( $post );
 
-	// bascially all child pages of the about or members pages + all the posts in the projects post type get the side menu
+	// bascially all child pages of the about, members or services pages + all the posts in the projects post type get the side menu
 	if ( is_page( $about_pg_id ) || in_array( $about_pg_id , $ancestors) )
 		$show_menu = 'About';
 	if ( is_page( $members_pg_id ) || in_array( $members_pg_id , $ancestors) )
 		$show_menu = 'Membership';
 	if ( is_singular( 'pauinn_project' ) || is_page( $programs_pg_id ) )
 		$show_menu = 'Projects';
+	if ( is_page( $services_pg_id ) || in_array( $services_pg_id , $ancestors) )
+		$show_menu = 'Services';
 
 	// yep, we should show a menu, modify the layout appropriately
 	if ( $show_menu != '' ) {
 		$content_class = 'span9 has-menu';
 		echo '<div class="internal-subnav span3 visible-desktop">';
 	}
 
-	// about and member pages and children get their respective page trees
-	if ( $show_menu == 'About' || $show_menu == 'Membership' ) {
-		$pg_id = ( $show_menu == 'About' ) ? $about_pg_id : $members_pg_id;
+	// about, member and services pages and children get their respective page trees
+	switch ( $show_menu) {
+		case "About":
+			$pg_id = $about_pg_id;
```

Comment:
These cases need `break` statements; I'll add them.

Reply:
Ah yes, thanks for catching that!

Reply:
https://github.com/INN/inn/pull/96/commits/ea51da4f674facbd6ee10b4ecda981714c50fe3b (if you haven't already)